projects/dashboards/src/persistence/dashboard-persistence.service.ts

```diff
@@ -15,12 +15,12 @@ import {
   providedIn: 'root'
 })
 export class DashboardPersistenceService {
-  private static readonly HYPERTRACE_LOCATION_NAMESPACE: string = '7f12784d-0bd1-4b0f-94da-cb8aa3b7fc9e';
+  // Private static readonly HYPERTRACE_LOCATION_NAMESPACE: string = '7f12784d-0bd1-4b0f-94da-cb8aa3b7fc9e';
```

Comment:
nit: can we convert `Private` to `private`

Reply:
Its a comment and there is a listing rule that's prevents starting comments with lowercase letters. This file is not needed in phase 1 so I think we can keep it as it is and once phase 1 PR is merged it would be fixed